### ISSUE TITLE
Examples not rendering for 'text/plain' content [Release 3.6.7]

### ISSUE THREAD

**Issue by Hydragyrum** [Mermade/widdershins#282]
**Describe the bug**
A clear and concise description of what the bug is. Reports containing a full reproduction recipe (command-line and input file gist) will get a **much** faster response.

Examples do not render when response content type is 'text/plain'

* [x] - I have checked that my input document is valid OpenAPI 2.0/3.0.x or AsyncAPI 1.x

**To Reproduce**
1. Command-line:
`widdershins --summary test.yaml -o test.md`

2. My input file: (please link to a gist)
```yaml
openapi: 3.0.2
info:
  title: "Test API"
  version: "1.0.0"
  contact:
    name: Contact Name
    email: contact@example.com
    url: https://example.com/
servers:
- url: https://example.com
  description: Test Server
tags:
  - name: Generic
    description: Generic API
paths:
  /:
    get:
      operationId: getStuff
      summary: Get something
      tags: [ 'Generic' ]
      responses:
        '200':
          description: OK
          content:
            text/plain:
              schema:
                type: string
              examples:
                "200 OK":
                  description: 200 OK
                  value: Hello World
```
Output markdown:
````markdown
---
title: Test API
language_tabs:
  - http: http
toc_footers: []
includes: []
search: true
highlight_theme: darkula
headingLevel: 2

---

<h1 id="test-api">Test API v1.0.0</h1>

> Scroll down for code samples, example requests and responses. Select a language for code samples from the tabs above or the mobile navigation menu.

Base URLs:

* <a href="https://example.com">https://example.com</a>

Email: <a href="mailto:contact@example.com">Contact Name</a> Web: <a href="https://example.com/">Contact Name</a> 

<h1 id="test-api-generic">Generic</h1>

Generic API

## Get something

<a id="opIdgetStuff"></a>

> Code samples

```http
GET https://example.com/ HTTP/1.1
Host: example.com
Accept: text/plain

```

`GET /`

> Example responses

> 200 OK

<h3 id="get-something-responses">Responses</h3>

|Status|Meaning|Description|Schema|
|---|---|---|---|
|200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|string|

<aside class="success">
This operation does not require authentication
</aside>
````

**Expected behavior**

After the `> 200 OK` line, we expect to see the example.

if the content type is set to `application/json`, for example, the following is shown:
````markdown
> 200 OK

```json
"Hello World"
```
````

**Screenshots**

For content type `application/json`
![image](https://user-images.githubusercontent.com/4928181/69948147-62781e80-14ef-11ea-953c-167eecb8b3d7.png)

For content type `text/plain`
![image](https://user-images.githubusercontent.com/4928181/69948196-7c196600-14ef-11ea-82f1-429cd41bf7be.png)

**Versions**:
- Node.js (note only LTS versions are supported): 
```
❯ node -v
v12.13.1
```

**Additional context**

**Comment by Hydragyrum** [comment]
I picked up the master branch from github and it seems to work on that. However, I have a CI server which does not have access to gitlab, and thus would still have the problem (v3.6.7)

**Comment by MikeRalphson** [comment]
Should be fixed in version `4.0.0`, which includes breaking changes (please see the `README.md` file).
